admin/src/User/User.tsx

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
if this is the index file of the User dirctory, call it index.tsx. That way your app will be able to require it with `import ... from './User';`

Reply:
Thanks for the tip